projects/vdk-core/src/vdk/internal/builtin_plugins/ingestion/ingester_base.py

```diff
@@ -439,6 +439,21 @@ def _payload_poster_thread(self):
                     for payload_dict in payload_obj:
                         self.__verify_payload_format(payload_dict=payload_dict)
 
+                    if ingestion_metadata:
+                        updated_dynamic_params: Optional[dict] = ingestion_metadata.pop(
+                            IIngesterPlugin.UPDATED_DYNAMIC_PARAMS, None
+                        )
+                        if updated_dynamic_params:
+                            destination_table = (
+                                updated_dynamic_params.get("destination_table")
```

Comment:
Both "destination_table" and "target" should be constants as well. They are used in more than one place for certain and must be the same.

Reply:
Fixed, thanks